nodes/index.adoc

```diff
@@ -17,6 +17,18 @@ Having stable and healthy nodes in a cluster is fundamental to the smooth functi
 In {product-title}, you can access, manage, and monitor a node through the `Node` object representing the node.
 Using the OpenShift CLI (`oc`) or the web console, you can perform the following operations on a node.
 
+Following are the components of a node responsible for maintaining the running of pods and providing the Kubernetes runtime environment.
+
+* *Container runtime*: The container runtime is the software that is responsible for running containers. Kubernetes supports several runtimes: Docker, containerd, cri-o, rktlet, and any implementation of the Kubernetes Container Runtime Interface (CRI).
+
+* *Kubelet*: Kubelet is the agent that runs on each node in the cluster. This agent ensures that the containers are running on the nodes as expected. Kubelet is responsible for relaying information to and from control plane service. Kubelet interacts with etcd store to read configuration details and write values. It communicates with the master component to receive commands and work. The kubelet process maintains the state of work and the node server. Kubelet manages network rules and port forwarding. The kubelet takes a set of PodSpecs that are provided through various mechanisms and ensures that the containers described in those PodSpecs are running and healthy. The kubelet manages containers that are created by Kubernetes only.
+
+* *Kube-proxy*: Kube-proxy is a proxy service which runs on each node and helps in making services available to the external host. It helps in forwarding the request to correct containers and is capable of performing primitive load balancing. A Kube-proxy ensures that the networking environment is predictable, isolated, and accessible. It manages pods on node, volumes, secrets, creating health checkup of a new container.
+
+* *DNS*: Cluster DNS is a DNS server which serves DNS records for Kubernetes services. Containers started by Kubernetes automatically include this DNS server in their DNS searches.
+
+image::openshift-nodes-overview.png[Nodes overview]
```

Comment:
In the graphic, should "master" be called "control plane" to match the descriptive text in the document and also follow the inclusive language guidelines? I am not sure if these two gray boxes represent nodes but if so, maybe they could be titled "Kubernetes control plane node" and Kubernetes worker nodes". It looks like the Kubernetes documentation also uses the "control plane" language also instead of "master."

Reply:
Ack. will follow up regarding the changes in the diagram